llvm/utils/mlgo-utils/mlgo/corpus/make_corpus.py

```diff
@@ -12,43 +12,38 @@
   --default_args="<list of space separated flags>"
 """
 
-from absl import app
-from absl import flags
-from absl import logging
+import logging
+import argparse
```

Comment:
can the imports be in alphabetical order, or does `black` not do that for us?

Reply:
Black doesn't do it automatically. I've put the script imports into alphabetical order.